packages/access-api/src/utils/html.js

```diff
@@ -140,30 +140,24 @@ export const PendingValidateEmail = ({ autoApprove }) => (
 export const ValidateEmail = ({ ucan, qrcode, email, audience }) => (
   <div class="fcenter">
     <img
-      src="https://web3.storage/android-chrome-512x512.png"
+      src="https://camo.githubusercontent.com/56c7b9e4143bd5db3e3e6d4ff03c077dd0a7f16b9f96f04d81b272caa9266b3e/68747470733a2f2f6261667962656961346c75756e73366467796d79356b617535726d377234717a72727a673663676c707a706f67757373707270793432636d636e342e697066732e7733732e6c696e6b2f773375702d6c6f676f2e706e67"
       height="80"
-      width="80"
     />
     <h1>Email Validated</h1>
     <p>{email} was confirmed. You may close this window.</p>
     <div class="box">
       <p>
-        During the beta period, uploads via w3up will only appear via the beta
-        APIs, and not on the web3.storage or NFT.Storage websites, even if the
-        associated email addresses are the same.
+        By registering with either the web3.storage or the NFT.Storage w3up beta, 
+        you agree to the respective Terms of Service 
+        (<a href="https://console.web3.storage/terms">web3.storage ToS</a>, 
```

Comment:
this will link to production TOS even when this page is served on staging, but in practice that seems like NBD